loan.rb

```diff
@@ -0,0 +1,15 @@
+module Loan
+    # country will get loan from INF if gdp>=5 and state is Developed
+    # country will get loan from WB if gdp>=4 and state is developed or developing
+    def will_get_loan(gdp, state)
```

Comment:
as per the method name it seems it will return boolean value, but it is printing type of loan. method name need to be improved